src/components/views/dialogs/InviteDialog.js

```diff
@@ -381,10 +379,9 @@ export default class InviteDialog extends React.PureComponent {
         return recents;
     }
 
-    _buildSuggestions(excludedTargetIds: string[]): {userId: string, user: RoomMember} {
+    _buildSuggestions(excludedTargetIds: Set<string>): {userId: string, user: RoomMember} {
         const maxConsideredMembers = 200;
         const client = MatrixClientPeg.get();
-        const excludedUserIds = [client.getUserId(), SdkConfig.get()['welcomeUserId']];
         const joinedRooms = client.getRooms()
```

Comment:
Collapse this down to `MatrixClientPeg.get()` please, to get rid of the `client` variable that isn't needed. It was only added for the linter.